### ISSUE TITLE
Thread Watcher presented file 'expansion' with user tags and ratings

### ISSUE THREAD

**Issue by alidan** [hydrusnetwork/hydrus#453]
Is it possible to have a user defined, though tags and ratings, presented for watchers?

my reasoning is a bit long so i put the end question at the top, but here is my use case. 

Ok, I'm doing culling due to entirely to many files, I mass downloaded 4chan threads I liked till I nearly filled a drive due to 'Ill get to it later' thought process
so I let watchers just hang around for entirely too long
I also like doing the culling from watchers due to being able to at least have some semblance of a theme for the files rather then just a random batch I get with search settings. 

However when I open presented files, I get 4 options, I get 
presented/new/all files/all + trash
So the only one I get a say in is presented, and it's been long enough I don't know if there are advanced options here. 
so I can have it present new files, already in db in inbox or in archive. 

ok, that helps but isnt what im looking for, if I was doing in depth 'lets deal with every everything about this image now' phase, sure, that may be enough. I am in a VERY rough cull, so I just look at the image, and make a snap 'do I like it' or 'get rid of it' and I rate them at the end with a 'Appealing Pass' rating, the thought is if I get to 10 while im culling, that image clearly is something I want to keep, the more stars the more obvious it is, the idea was over time I would rate everything, then search the rated ones and star them further and further, till they are clearly image I want to keep forever. 

now, while i'm looking at my current opened pages and see some images with 4-8 stars, my process is clearly working, however this process was intended for down the line, not at the cull. 

An expansion so I can have a user defined 'anything without appealing pass rating' so I only see files without it. 

now, lets be clear, new files would effectively work this way, however I have so many files in thread watchers going back so long, on top of before the thread watcher revamp happened, that even the old files in many of these have likely yet to hit a cull pass.

**Comment by imtbl** [comment]
While not exactly the same, #467 could be used in a similar manner to achieve the same thing and would generally be a saner solution UX-wise.

If you still think this feature request is necessary/valid even if #467 gets implemented, please rephrase it into something more concise that's easier to read and understand and reopen.

**Comment by alidan** [comment]
mine can say closed and I can reword it better, but I have absolutely no idea what #467 is even asking for, before I make a new feature request hopefully better explaining what I want, able to help me with that?

**Comment by imtbl** [comment]
> Tag searching and collections in downloader/import/publish pages

Does that really not give it away?

**Comment by alidan** [comment]
not really, is it asking for collections as in image sets, searching tags inside a set of files, or how it would be saner from the ux side of things. 

I mean, if its a soft search as in lets say I pull up a watcher
I then search for 'non mcguffin tag' and it soft hides them from view till the search is cleared, that I could understand and get behind, would be cumbersome for my use but I could load up several threads at once and mitigate that issue. If its asking for just a normal search within the image set and its not reversible or likely wouldn't be, then I think my ask has more practical applications, but like I said, I will re word it hopefully with pictures to better explain.

**Comment by imtbl** [comment]
#467 is asking for the same searching/collection feature set you have on regular file pages to be available on downloader/import/publish pages as well, except with the search being limited to the files downloaded on that page. Of course the search would be reversible (as you can also modify the search on regular pages).

It's a good approach UX wise because it's an existing feature that could be made consistent across all page types, so the user knows what to expect.

Like I said, it's not a 1:1 replacement for your proposal, but since both allow you to filter files, you can basically achieve the same thing.

Personally, I don't think your idea is bad (and could even be used in conjunction with #467). But yes, please reword it to be more concise and to the point. It's currently very hard to understand what you are trying to communicate in the feature request.

**Comment by alidan** [comment]
Thanks, had no idea that was even a feature, if this got put into download pages or watchers, I think that would be perfect for my use, especially if I can set it once and forget it. this would essentially do exactly what I want rather than my thought of having haveing a user defined show, "show all X" button. 

that said, if possible I have a question

Is collections a new feature as in not fully fleshed out yet, because I have a few thoughts on what it currently does and works.
